Source/core/Netlink.h

```diff
@@ -396,7 +437,7 @@ namespace Core {
 
     public:
         SocketNetlink(const Core::NodeId& destination)
-            : SocketDatagram(false, destination, Core::NodeId(), 512, 1024)
+            : SocketDatagram(false, destination, Core::NodeId(), 4096, 8192)
```

Comment:
Did you get the sizes from a spec? Sending 4K and Receiving 8K? sounds big... Maybe good to put the reference here where you found this..

Reply:
I don't think they are limited by spec, however, in practice, it is limited to page size. Moreover, it is limited to 8k (https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tree/include/linux/netlink.h?h=linux-4.9.y#n112). I limited send buffer to 4k as we control it and can always serve a more restricted 4k. I will add it in the comment :)

Reply:
In practice I fount multiple cases where message was >4k and splitter into 2 ~3.5k  (as rpi page size is 4k). I think limiting it to 4k would be a little bit on dangerous size